opendm/rollingshutter.py

```diff
@@ -0,0 +1,41 @@
+from opendm import log
+
+# Make Model (lowercase) --> readout time (ms)
+RS_DATABASE = {
+    'dji phantom vision fc200': 74, # Phantom 2
+    
+    'dji fc300s': 33, # Phantom 3 Advanced
+    'dji fc300s': 33, # Phantom 3 Standard
+    'dji fc300x': 33, # Phantom 3 Professional
+
+    'dji fc330': 33, # Phantom 4
+    'dji fc6310': 33, # Phantom 4 Professional
+
+    'dji fc350': 30, # Inspire 1
+
+    'gopro hero4 black': 30 # GoPro Hero 4 Black
+
+    # Help us add more! Open a pull request.
```

Comment:
Any clues on how to determine this for a different model?

Reply:
We're trying to figure that out: https://community.opendronemap.org/t/phantom-4-photo-request-rolling-shutter/11777

Reply:
Thanks @pierotofy -- this is very exciting!

Reply:
There we go: https://github.com/OpenDroneMap/RSCalibration